deps/rabbit/test/classic_queue_SUITE.erl

```diff
@@ -83,6 +83,8 @@ leader_locator_client_local(Config) ->
                                       {<<"x-queue-leader-locator">>, longstr, <<"client-local">>}])),
          {ok, Leader0} = rabbit_ct_broker_helpers:rpc(Config, Server, rabbit_amqqueue, lookup, [rabbit_misc:r(<<"/">>, queue, Q)]),
          Leader = amqqueue:qnode(Leader0),
+         ?assertEqual([{leader, Leader}, {members, [Leader]}],
```

Comment:
This test fails in CI.

Reply:
This assertion generally does not belong to this test. It does not test info items returned by CQ processes.